go.mod

```diff
@@ -1,99 +1,100 @@
 module github.com/VictoriaMetrics/victorialogs-datasource
 
-go 1.21.6
+go 1.23.1
```

Comment:
please upgrade ci jobs as well

Reply:
or just set it in CI jobs like it's described here
https://github.com/actions/setup-go?tab=readme-ov-file#getting-go-version-from-the-gomod-file

Reply:
fixed